infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/devfile/KubernetesComponentToWorkspaceApplierTest.java

```diff
@@ -188,10 +194,10 @@ public void shouldUseReferenceContentAsRecipeIfPresent() throws Exception {
   public void shouldProvisionEnvironmentWithCorrectRecipeTypeAndContentFromOSList()
       throws Exception {
     // given
-    String yamlRecipeContent = getResource("petclinic.yaml");
+    String yamlRecipeContent = getResource("devfile/petclinic.yaml");
     doReturn(toK8SList(yamlRecipeContent).getItems()).when(k8sRecipeParser).parse(anyString());
     ComponentImpl component = new ComponentImpl();
-    component.setType(OPENSHIFT_COMPONENT_TYPE);
+    component.setType(KUBERNETES_COMPONENT_TYPE);
```

Comment:
If this test case no longer has anything to do with OpenShift the name should be changed.

Reply:
I think it shold be reverted since it duplicates another test fully now